src/main/java/hmim/eteam/rest/backend/controller/AuthController.java

```diff
@@ -63,7 +67,11 @@ public AuthToken generateToken(SiteUser siteUser) {
             --triesLeft;
         } while (authTokenRepository.existsById(id) || triesLeft == 0);
 
-        return authTokenRepository.save(new AuthToken(id, siteUser, new Date()));
+        Calendar calendar = Calendar.getInstance();
+        calendar.setTime(new Date());
+
+        calendar.add(Calendar.MINUTE, SESSION_LENGTH_IN_MINUTES);
+        return authTokenRepository.save(new AuthToken(id, siteUser, calendar.getTime()));
```

Comment:
consider migrating `AuthTokenRepository` to use `Calendar`

Reply:
It's not supported by JPA, as I know.